modules/core/include/opencv2/core/cuda.hpp

```diff
@@ -569,18 +569,18 @@ class CV_EXPORTS HostMem
     hardware. Laptops often share video and CPU memory, so address spaces can be mapped, which
     eliminates an extra copy.
      */
-    GpuMat createGpuMatHeader() const;
+    CV_WRAP GpuMat createGpuMatHeader() const;
 
     // Please see cv::Mat for descriptions
-    bool isContinuous() const;
-    size_t elemSize() const;
-    size_t elemSize1() const;
-    int type() const;
-    int depth() const;
-    int channels() const;
-    size_t step1() const;
-    Size size() const;
-    bool empty() const;
+    CV_WRAP bool isContinuous() const;
+    CV_WRAP size_t elemSize() const;
+    CV_WRAP size_t elemSize1() const;
+    CV_WRAP int type() const;
+    CV_WRAP int depth() const;
+    CV_WRAP int channels() const;
+    CV_WRAP size_t step1() const;
```

Comment:
`step1` is wrapped, but `step` is not.

Same for GpuMat`

Reply:
`step` is now wrapped in both classes.